src/SidePanel/ChannelView.tsx

```diff
@@ -42,45 +40,33 @@ const DelaySlider = ({
     const range = { min: 20, max: 20000, decimals: 0 };
     const isTransmitterPane = useSelector(getIsTransmitterPane);
     return (
-        <div className="slider-container">
-            <FormLabel htmlFor="sweep-delay-slider">
-                {isTransmitterPane
+        <NumberInput
+            label={
+                isTransmitterPane
                     ? 'Transmit period per channel'
-                    : 'Receive period per channel'}
-                <NumberInlineInput
-                    value={currentValue}
-                    range={range}
-                    disabled={isRunning}
-                    onChange={value =>
-                        changedFunc(isRunning ? currentValue : value)
-                    }
-                />{' '}
-                ms
-            </FormLabel>
-            <Slider
-                id="sweep-delay-slider"
-                values={[currentValue]}
-                disabled={isRunning}
-                onChange={[
-                    value => changedFunc(isRunning ? currentValue : value),
-                ]}
-                range={range}
-            />
-        </div>
+                    : 'Receive period per channel'
+            }
+            showSlider
+            minWidth
+            range={range}
+            value={currentValue}
+            unit="ms"
+            disabled={isRunning}
+            onChange={value => changedFunc(isRunning ? currentValue : value)}
+        />
     );
 };
 
-DelaySlider.propTypes = {
-    isRunning: PropTypes.bool.isRequired,
-    currentValue: PropTypes.number.isRequired,
-    changedFunc: PropTypes.func.isRequired,
-};
-
 interface Props {
     paneName: 'transmitter' | 'receiver';
 }
 
-const ChannelView: React.FC<Props> = ({ paneName }) => {
+const bleChannelsValues = [
```

Comment:
Do you want to phase out the shared export?

Reply:
I think it is a bot meaning less what we share from shared ... what do you think?

Reply:
Not sure I understand. But it was just a question I think it's fine either way.